docs/dashboards/custom-component.md

```diff
@@ -386,3 +386,57 @@ Dashboards.board('container', {
     }]
 });
 ```
+
+
+## Custom Component with data from the Connector
+
+The example below shows how to program a custom component that fetches data from the Connector processes and displays it on the Dashboard.
```

Comment:
that fetches data from the Connector processes and displays it  -> comma between Connector and processes